pkg/vegeta/report.go

```diff
@@ -86,5 +86,22 @@ decode:
 		return json.Marshal(jsonReportResponse)
 	}
 
+	if format == TextFormat {
+		buf, err = addID(buf, id)
```

Comment:
```suggestion
		bid := []byte(fmt.Sprintln("Id", id))
		buf = bytes.NewBuffer(append(bid, buf.Bytes()...))
```
I think instead of using `ReadFrom` method, we can directly append the required bytes, it would be more fast and would be far from any errors.

Reply:
I used a method in order to make the change testable.

Reply:
I was not concerned about the `addID` method, I was talking about the `ReadForm` function, which returns error.
Tests are always good, and must be included. 👍